roadmap/EF2023.md

```diff
@@ -0,0 +1,142 @@
+# EF JavaScript Team Roadmap 2023
+
+This document reflects our initial planning for the respective year and represents
+the roadmap we have submitted internally within the EF.
+
+Things in the Ethereum ecosystem move quickly, so please note that we will likely deviate
+substantially from this initial roadmap throughout the year.
+
+See our "live" [roadmap](./README.md) document for an up-to-date overview on what we
+are currently up to.
+
+## Tracks
+
+The roadmap is divided into the following "tracks", identifying our main focus areas
+for the year.
+
+### DevEx & VM
+
+The [EthereumJS](https://github.com/ethereumjs/ethereumjs-monorepo) libraries are often used in a development context, serving e.g. as an integral part of main Ethereum developer tools or as building blocks for browser UIs of decentralized applications (dApps). Improving the experience when working with our libraries is therefore a crucial part of our work, with the VM playing a special role here due to the central place it takes in development tools like Hardhat, Truffle or Remix.
+
+### Hardforks & EIPs
+
+Implementing hardforks and providing early-on access to the added or changed functionality in development tools takes a reoccurring share in our work. We also do selected implementations of dedicated EIPs not yet "hardfork-settled" where it makes sense and help on improving EIP specifications and tests.
+
+### Research, Client & Testnets
+
```

Comment:
I don't know where this fits in the timeline but I would really love to get our browser client up and running again.  Now that CL light clients are a thing, it would be really cool to pair that with an EL ethJS client running in something like light client mode.  We could pair it with the Lodestar light client and maybe even integrate the Ethers State Manager somehow to give us comprehensive access to state data.

Reply:
Have added for Q1.